lib/services/suppressions-service.js

```diff
@@ -208,11 +216,11 @@ class SuppressedViolationsManager {
     /**
      * Updates the suppressions file.
      * @param {SuppressedViolations} suppressions The suppressions to save.
-     * @returns {void}
+     * @returns {Promise<void>}
      * @private
      */
-    save(suppressions) {
-        fs.writeFileSync(
+    async save(suppressions) {
+        await fs.promises.writeFile(
             this.suppressionsLocation,
             JSON.stringify(suppressions, null, 2)
```

Comment:
This doesn't have to be an async function
```suggestion
    ave(suppressions) {
        return fs.promises.writeFile(
            this.suppressionsLocation,
            JSON.stringify(suppressions, null, 2)
```